templates/security/valide_authentification.html.twig

```diff
@@ -12,7 +12,7 @@
 
 
                 <div class="form-group">
-                    <label for="qrCode">Code de validation</label>
+                    <label for="qrCode">Validate code</label>
                     <input type="text" id="qrCode" name="qrCode" class="form-control" required="required" />
                 </div>
                 <input type="submit" class="btn btn-primary" value="Confirmer" />
```

Comment:
I think you forget some french translations :-)

Reply:
shit